parsing.indexing/test/unit/src/org/netbeans/modules/parsing/impl/indexing/ScanStartedTest.java

```diff
@@ -497,7 +497,9 @@ public void publish(LogRecord record) {
                 msg != null &&
                 msg.startsWith("scanStarting:") &&  //NOI18N
                 ie.first().equals(record.getParameters()[0])) {
-                throw ie.second();
+                RuntimeException exception = internalException.second();
```

Comment:
The exception should be taken from "ie" local var anyway so the reported exception from .second() is consistent with ie.first().

It could be better to remove the handler instance added in ``setUp()`` by  `logger.removeHandler(this.handler);` in `tearDown`. Handler from preceding testcases runnin in the same VM pile up. Maybe it wouldn't be even necessary to clean handler state then.